sdlf-utils/pipeline-examples/datalake-workload-management/utils/bootstrap.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -Eeuox pipefail
+
+pflag=false
+rflag=false
+bflag=false
+
+DIRNAME=$(pwd)
+
+usage () { echo "
+    -h -- Opens up this help message
+    -p -- AWS profile
+    -r -- AWS Region to deploy to (e.g. eu-west-1)
+    -b -- foundational Bucket name for artifacts
+"; }
+options=':p:r:b:h'
+while getopts $options option
+do
+    case "$option" in
+        p  ) pflag=true; PROFILE=${OPTARG};;
```

Comment:
We use -s for the shared cicd profile and -t for the child. Can you please modify this to align with the standard deploy script?

Reply:
The sdlf-wlm-integration doesnt require any deployment script changes or additions based on new changes. Standalone solution is kept as is